app/Http/Controllers/LiquidationController.php

```
@@ -0,0 +1,81 @@
+namespace App\Http\Controllers;
+
+use App\Enums\PaymentRequestType;
+use App\Enums\RequestStatuses;
+use App\Http\Requests\LiquidationFormRequest;
+use App\Models\PaymentRequest;
+use App\Models\StakeHolder;
+use Illuminate\Http\Request;
+
+class LiquidationController extends Controller
+{
+    public function index(Request $request)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (undefined)

(UnusedFormalParameter)